Sources/Serveable.swift

```diff
@@ -0,0 +1,10 @@
+
+/// Serves your order to a local or a remote bar
+
+public protocol Serveable {
+
+    associatedtype S
+
+    func serve(order: Order, delivery: (Delivery<S>)->())
```

Comment:
I'm a bit scared that we are moving away from readability...

When I read `Bar.serve(order)` and I read `Fare.perform(request)`, I'm reading the last one more easily? The first one is like 👙(to much magic) to me. Not?